src/Stripe.net/Enums/Billing.cs

```diff
@@ -4,6 +4,7 @@ namespace Stripe
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
 
+    // Deprecated. Please use CollectionMethod
```

Comment:
Regular comments are not visible to users (lest they actually read the library's source code). You should change this to use an [XML comment](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/) and/or the `Obsolete` attribute (probably both).

Reply:
Thanks! Updated.